.circleci/config.yaml

```diff
@@ -0,0 +1,14 @@
+version: 2
```

Comment:
I'm less familiar with Circle CI setup so gently paging @rghetia and @pavolloffay for a review.